can_tools/scrapers/official/SC/sc_demographics.py

```diff
@@ -7,7 +7,7 @@
 from can_tools.scrapers.official.base import StateDashboard
 
 
-class SCVaccineDemographics(StateDashboard):
+class SCVaccineRace(StateDashboard):
```

Comment:
SC changed the structure of their dashboard, so this updates the scraper to match (and pulls in ethnicity values)